plugins/modules/redfish_command.py

```diff
@@ -859,14 +886,19 @@ def main():
         for command in command_list:
             if command == "SimpleUpdate":
                 result = rf_utils.simple_update(update_opts)
+                if 'update_status' in result:
+                    redfish_facts['update_status'] = result['update_status']
+            elif command == "PerformRequestedOperations":
+                result = rf_utils.perform_requested_update_operations(update_opts['update_handle'])
 
     # Return data back or fail with proper message
     if result['ret'] is True:
         del result['ret']
         changed = result.get('changed', True)
         session = result.get('session', dict())
         module.exit_json(changed=changed, session=session,
-                         msg='Action was successful')
+                         msg='Action was successful',
+                         redfish_facts=redfish_facts)
```

Comment:
Is this addition okay? Most commands will not have any output to redfish_facts, and it doesn't seem normal from a "command" perspective to have output data like this, but I couldn't think of other ways to encapsulate this information.

Reply:
This is generally OK. Having return values with more or less complex content is normal for modules :)

I'm not sure the name `redfish_facts` fits. Maybe use something more general like `return_values` instead? `_facts` sounds a bit like `ansible_facts`, and these are not Ansible facts.

You should also document this return value in https://github.com/ansible-collections/community.general/blob/e152a1556fb1ade1d25ed339619d21b1951884ea/plugins/modules/redfish_command.py#L628-L634.

Reply:
Changed the name to `return_values` and added documentation about it.